posthog/version.py

```diff
@@ -1 +1,9 @@
 VERSION = "1.27.0"
+
+try:
+    import git
+
+    repo = git.Repo(search_parent_directories=True)
+    GIT_SHA = repo.head.object.hexsha
+except Exception:
+    GIT_SHA = "Unavailable"
```

Comment:
How sure are we that the deployments we care about will be cloned with Git and will have Git available? For instance, this doesn't work when I run PostHog in `docker-compose`. Will it work via Kubernetes?
For development setups we don't need this, because the DEBUG toolbar already gives this info.

An alternative way that would work everywhere could be having a hardcoded fallback `GIT_SHA`. That could be done with an Actions workflow that would update the value for every commit to `master`, though that _would_ be really intensive in the number of such commits.